pandas/core/indexing.py

```diff
@@ -189,8 +187,9 @@ def _get_setitem_indexer(self, key):
         if isinstance(key, range):
             return self._convert_range(key, is_setter=True)
 
+        ax_num = self.axis or 0
```

Comment:
just use axis

Reply:
there is already an `axis` in the namespace

Reply:
then the other one is wrong (and likely should be called ax), consistency is so important here

Reply:
changed